media/js/libs/fontfaceobserver/fontfaceobserver-standalone.js

```diff
@@ -0,0 +1,41 @@
+/*
+
+https://github.com/bramstein/fontfaceobserver
+v1.4.5
+
+Copyright (c) 2014 - Bram Stein
```

Comment:
@gerv I could use a 3rd opinion on the inclusion of this library if you have a moment. The original repo is here: https://github.com/bramstein/fontfaceobserver I copied only fontfaceobserver-standalone.js from the repo but the file did not include the license so I added the license into the file. I need to double check that 1) the license is compatible with MDN and 2) I am adhering to it.